domains/drubico.json

```
@@ -0,0 +1,9 @@
+{
+  "description": "Portfolio's Diego Rubi",
+  "owner": {
+    "username": "drubico"
+  },
+  "record": {
+    "CNAME": "https://drubico.github.io/portfolio"
```

CNAMEs cannot have https or /

```suggestion
    "CNAME": "drubico.github.io"
```